homeassistant/components/airnow/__init__.py

```diff
@@ -146,4 +168,30 @@ async def _async_update_data(self):
         data[ATTR_API_AQI_DESCRIPTION] = max_aqi_desc
         data[ATTR_API_POLLUTANT] = max_aqi_poll
 
+        if self.forecast_attrs:
+            try:
+                forecast = await self.airnow.forecast.latLong(
+                    latitude=self.latitude,
+                    longitude=self.longitude,
+                    date=datetime.datetime.now() + datetime.timedelta(days=1),
+                    distance=self.distance,
+                )
+
+            except (AirNowError, ClientConnectorError) as error:
+                raise UpdateFailed(error) from error
+
+            if not forecast:
+                raise UpdateFailed("No forecast data was returned from AirNow")
+
+            data[SENSOR_AQI_ATTR_FORECAST] = [
```

Comment:
For consistency with the rest of the integration, this should probably just store the raw `forecast` variable returned from AirNow in an `ATTR_AQI_FORECAST` key, and do the below processing in sensor.py.